rfc-012-makefile-standards.md

```diff
@@ -0,0 +1,64 @@
+# Makefile standards
+
+## Summary
+
+A set of standards for Makefiles across all projects
+
+## Problem
+
+Having a standard set of commands that can be run across all projects for common tasks
+allows people to get projects up and running without having to spend time looking into
+how things are set up to know what specific commands to run.
+
+
+## Proposal
+
+All projects MUST have a Makefile.
+
+All projects must have relevant [`PHONY` targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)
+
+**All** projects MUST have the commands:
+- `make test`
+  - Runs the tests for the project
+  - This SHOULD run the tests in a 'watch' mode (e.g. [Guard](https://github.com/guard/guard))
```

Comment:
Should this be a separate task? `make test-watch` rather than the default test behaviour.